pkg/reconciler/v1alpha1/broker/resources/ingress.go

```diff
@@ -58,6 +58,7 @@ func MakeIngress(args *IngressArgs) *appsv1.Deployment {
 			Template: corev1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
 					Labels: ingressLabels(args.Broker),
+					// TODO: Remove this annotation once all channels stop using istio virtual service
```

Comment:
Link to the GH issue (#294 I think).

Reply:
Just mention issue number in comment or is there a syntax to link to github issue?

Reply:
My preference is a full URL, so that it is easy to click in any environment, not just GitHub itself.